plugin/index.js

```diff
@@ -232,6 +234,27 @@ class DashboardPlugin {
     // Get the **full** stats object here for `inspectpack` analysis.
     const statsToObserve = statsObj.toJson();
 
+    // Truncate off non-included assets.
+    const { includeAssets } = this;
+    if (includeAssets.length) {
```

Comment:
correct me if I'm wrong but includeAssets is always an array, so this condition is needed only to double-check? 🤔

Reply:
It's meant to only apply the filter of `includeAssets` if it has a length of 1 or more, as we don't apply any filtering if the option is unset or `includeAssets === []`